### ISSUE TITLE
Incompatible with Upcoming Events mod

### ISSUE THREAD

**Issue by Xymanek** [WOTCStrategyOverhaul/CovertInfiltration#56]
We spawn our custom event queue items so that mod's overrides do nothing. However there is no conflict either

**Comment by robojumper** [comment]
As in "Upcoming Events in Hours"? We should include that functionality, it seems quite convenient.

**Comment by Xymanek** [comment]
Yep, was thinking the same thing. I was also thinking to add MCM support and option to turn this off in case the player doesn't want it.
